### PR TITLE
PRL-3772 - Apply for HWF guidance

### PR DESCRIPTION
**Pull request by hcarslaw**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the keep-helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  line5: 'Enter your help with fees reference number',
+  onlyContinue: 'Continue',
+  cancel: 'Cancel',
+};
+
+export const cy: typeof en = {
+  title: 'Apply for help with fees (welsh)',
+  applyBefore: 'You must apply for help with fees before submitting your application. (welsh)',
+  nextSteps: 'Next steps (welsh)',
+  line1:
```
Comment (vivek-sekhar) [inline src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts:26]: Please use arrays for showing the steps. do not split lines.

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts (hunk 1 above)
Comment (hcarslaw) [reply src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts]: Changed to use arrays and whole lines

File: src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+import { FormContent } from '../../../../app/form/Form';
+import { getApplicationDetails } from '../../../../steps/application-within-proceedings/utils';
+import { interpolate } from '../../../../steps/common/string-parser';
+import { getCasePartyType } from '../../../../steps/prl-cases/dashboard/utils';
+
+export const en = {
+  title: 'Apply for help with fees',
+  applyBefore: 'You must apply for help with fees before submitting your application.',
+  nextSteps: 'Next steps',
+  bulletPoints: [
```
Comment (vivek-sekhar) [inline src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts:11]: change it to listItems.

File: src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    classes: 'govuk-!-margin-left-3',
+    href: '/',
+    text: l => l.cancel,
+  },
+};
+
+const generateBulletPoints = (translations, applicationDetails) => {
+  const bulletPoints: string[] = [];
+
+  translations.bulletPoints.forEach(bulletPoint => {
```
Comment (vivek-sekhar) [inline src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts:57]: use map constructor

File: src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    caseData.caseTypeOfApplication,
+    partyType,
+    content.language,
+    request.session.applicationSettings
+  );
+
+  return {
+    ...translations,
+    form,
+    bulletPoints: generateBulletPoints(translations, applicationDetails),
```
Comment (vivek-sekhar) [inline src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts:88]: you can directly run the map constructor.  No need of a separate method.

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts (hunk 4 above)
Comment (hcarslaw) [reply src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts]: Changed to use map

File: src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts (hunk 3 above)
Comment (hcarslaw) [reply src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts]: Changed to use map

File: src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts (hunk 2 above)
Comment (hcarslaw) [reply src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts]: Changed

**Review by vivek-sekhar — APPROVED** [review]

**Pull request merged**